src/app/calculator/utilities/electricity-reduction/electricity-reduction-form/electricity-reduction-form.component.ts

```diff
@@ -28,6 +28,8 @@ export class ElectricityReductionFormComponent implements OnInit {
   @Input()
   selected: boolean;
 
+  userSelectedHP: boolean;
```

Comment:
I'm not sure a userSelectedHP class property is needed. One sign of this:  the property changes state everywhere the form does and you've already made it part of the ElectricityReductionData that the form tracks. Either make changes here or keep this in mind for future